CH4_PoolDilution.html

```diff
@@ -11,7 +11,7 @@
 
 <meta name="author" content="Kendalynn A. Morris" />
 
-<meta name="date" content="2022-10-14" />
+<meta name="date" content="2022-10-27" />
```

Comment:
We should remove this html file from Git.

Reply:
This is probably because it was committed some time in the past, so git keeps tracking it. I think that fixing it will require a trip to the command line:
```
git rm --cached CH4_PoolDilution.html
```
Happy to handle this if you'd prefer.

Reply:
That would be great, I looked into this and it's not within my current skill set. Obviously this can wait until Monday though. Thank you!

Reply:
See https://github.com/COMPASS-DOE/grossMethane/pull/26